packages/devtools-reps/src/reps/function.js

```diff
@@ -51,7 +55,23 @@ function getTitle(props, grip) {
 
 function summarizeFunction(grip) {
   let name = grip.userDisplayName || grip.displayName || grip.name || "";
-  return cropString(name + "()", 100);
+  return cropString(name, 100);
+}
+
+function renderParams(props) {
+  const {
+    parameterNames = []
+  } = props;
+
+  return parameterNames
+    .filter(param => param)
```

Comment:
This is in `previewFunction` , I wonder if we should keep this.
From the debugger, can `parameterNames` have falsy values ?